docs/UserGuide.md

```diff
@@ -575,12 +592,45 @@ Format: `help`
 <div markdown="span" class="alert alert-primary">
   :bulb: <strong>Tip:</strong> You can press Esc key to close the help window!
 </div>
+<br/>
+
+Example:
+
+1. Suppose you can't remember the commands. Fret not, executing `help` will generate a new window containing the link to your user guide!
```

Comment:
*the user guide? since your implies the user owns the guide?

Reply:
fixed